pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -320,6 +322,7 @@ func newTestController(initialObjects ...runtime.Object) (*daemonSetsController,
 		informerFactory.Core().V1().Pods(),
 		informerFactory.Core().V1().Nodes(),
 		clientset,
+		flowcontrol.NewBackOff(50*time.Millisecond, 500*time.Millisecond),
```

Comment:
I think you want to use NewFakeBackOff here and provide a clock

Reply:
and provide the same clock to the daemonset controller and use it instead of time.Now()

Reply:
done